packages/flame/test/game/camera_and_viewport_test.dart

```diff
@@ -128,9 +128,11 @@ void main() {
       final canvas = MockCanvas();
       game.render(canvas);
       expect(
-        canvas.methodCalls.where((e) => e.startsWith('translate')),
+        canvas.methodCalls.where(
+          (e) => e.startsWith(RegExp('translate|transform')),
+        ),
         [
-          'translate(0.0, 0.0)', // camera translation
+          'transform(1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0)', // camera translation
```

Comment:
sadly these are not as simple to comprehend now, but I guess it's the price of progress haha

Reply:
Indeed, completely impossible to grasp from just a glance. We could translate the transform into translate and scale in the mock too I guess, but that also has its downsides.